azure-iot-device/tests/provisioning/test_provisioning_device_client_factory.py

```diff
@@ -0,0 +1,63 @@
+# -------------------------------------------------------------------------
```

Comment:
>------------------------------------------------------------------------- [](start = 2, length = 73)

this file should not be there right ?

Reply:
I have no idea where this came from. If you say it shouldn't, I'll delete it.

---
In reply to: [290989784](https://github.com/Azure/azure-iot-sdk-python-preview/pull/101#discussion_r290989784) [](ancestors = 290989784)